DOCS-6015-fix typos (#272)
mulesoft/docs-api-manager: modules/ROOT/pages/manage-exchange-api-task.adoc

include::_attributes.adoc[]
endif::[]

> To be able to manage your API that you created using Design Center or any other application, you must first import it to Anypoint Exchange (Exchange). After you create the API in API Manager by importing it from Exchange, you can modify the API version, status, and other information. After saving the configuration, you can deploy the API.

Reviewer: ```suggestion
To manage an API that you create, either using Design Center or any other application, you must first import the API to Anypoint Exchange (Exchange). After you create the API in API Manager by importing it from Exchange, you can modify the API version, status, and other information. After saving the configuration, you can deploy the API.
```
This chronology is confusing:
You create an API.
To manage it, you import to Exchange.
Then you create it in API Manager by importing it from Exchange to manage it? You have already created it.
Where is the user working: in Exchange or API Manager?
Are you saying...:
1. Create an API
2. Import to Exchange.
3. Export to API Manager.
4. In API Manager, modify.
5. Save.
6. Deploy to ...where?

Reviewer: Is this illustrated?

Reviewer: Looking at the title (which should be fixed to use this capitalization: Create an API Instance by Import from Exchange), I think what you mean is this:
```suggestion
To manage the API that you created using Design Center or any another application, you must first import it to Anypoint Exchange (Exchange). Then you import the API to API Manager, from which you can modify the API version, status, and other information. After saving your changes, deploy the API.
```
Deploy to...?
Also, does this mean that the original API still exists in Exchange, and the modified one in API Manager?

Author: Done! Thanks!

*Task Prerequisite*

> Before you you perform this procedure, ensure that you have the access permissions to an API in your organization's Exchange.

Reviewer: ```suggestion
Before you import the API from Exchange to API Manager and modify that API instance to suit your needs, ensure that you have the proper API access permissions in your organization's Exchange instance.
```

Reviewer: As indicated in earlier edits, I don't think we want to use inline headings like the one you use for prerequisites. 
But Mysti will clarify with her template at some point.

Author: Done! Thanks!
